components/AddEventForm/index.js

```diff
@@ -101,13 +102,13 @@ export default function AddEventForm({ onSubmit, appointment }) {
           type="date"
           name="date"
           defaultValue={appointment ? appointment.date : null}
-          required
+          required={selectedType === "wishlist" ? false : true}
```

Comment:
can be shortened, though it is up for discussion whether this causes your code to be less readable:

```suggestion
          required={!(selectedType === "wishlist")}
```

Reply:
Thanks, I implement it